pkgs/by-name/i3/i3ipc-gjs/package.nix

```
@@ -0,0 +1,47 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  autoconf,
+  automake,
+  gjs,
+  i3ipc-glib,
+  pkg-config,
+  which,
+}:
```

```suggestion
{ stdenv
, lib
, fetchFromGitHub
, autoconf
, automake
, gjs
, i3ipc-glib
, pkg-config
, which
}:
```
See https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#syntax:

> but if they don't fit on one line they're written as:
> 
> { arg1, arg2, arg3
> , arg4, ...
> , # Some comment...
>   argN
> }: